conda/environments/cudf_dev_cuda11.0.yml

```diff
@@ -60,3 +62,4 @@ dependencies:
       - git+https://github.com/dask/dask.git@master
       - git+https://github.com/dask/distributed.git@master
       - git+https://github.com/python-streamz/streamz.git
+      - pyorc
```

Comment:
for reference: conda-forge/staged-recipes#12937

Reply:
This has since been added. Submitted PR ( https://github.com/rapidsai/cudf/pull/11855 ) to use conda-forge